xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1686,7 +1686,7 @@ core.register.invalidUsername=Invalid username provided. Please use only letters
 core.register.mailSenderWronglyConfigured=The user has been created but the validation email has not been sent. Please check the Mail Sending Configuration and consider recreating the user.
 core.register.invalidCaptcha=Incorrect CAPTCHA answer.
 core.register.registerFailed=Registration has failed due to unknown reasons. (Error code: {0})
-core.register.successful.welcome=Welcome {0}
+core.register.successful.welcome=Welcome {0} {1}!
```

Comment:
@surli after the issue last time when updating a translation key.
Can you confirm that it's indeed unsafe to perform this change and that it's better to introduce a new key?